arrow-libs/optics/arrow-optics/src/commonMain/kotlin/arrow/optics/dsl/index.kt

```diff
@@ -10,38 +10,28 @@ import arrow.optics.typeclasses.Index
  * DSL to compose [Index] with a [Lens] for a structure [S] to focus in on [A] at given index [I]
  *
  * @receiver [Lens] with a focus in [S]
- * @param ID [Index] instance to provide a [Optional] to focus into [S] at [I]
+ * @param idx [Index] instance to provide a [Optional] to focus into [S] at [I]
  * @param i index [I] to focus into [S] and find focus [A]
  * @return [Optional] with a focus in [A] at given index [I]
  */
-public fun <T, S, I, A> Lens<T, S>.index(ID: Index<S, I, A>, i: I): Optional<T, A> = this.compose(ID.index(i))
-
-/**
- * DSL to compose [Index] with a [Prism] for a structure [S] to focus in on [A] at given index [I]
- *
- * @receiver [Prism] with a focus in [S]
- * @param ID [Index] instance to provide a [Optional] to focus into [S] at [I]
- * @param i index [I] to focus into [S] and find focus [A]
- * @return [Optional] with a focus in [A] at given index [I]
- */
-public fun <T, S, I, A> Prism<T, S>.index(ID: Index<S, I, A>, i: I): Optional<T, A> = this.compose(ID.index(i))
+public fun <T, S, I, A> Lens<T, S>.index(idx: Index<S, I, A>, i: I): Optional<T, A> = this.compose(idx.index(i))
```

Comment:
A `Lens` will be an `Optional`, right? So can we omit this syntax like we do for `nullable`?

Reply:
Done in the new commit